src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +624,121 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
+
+```bash
+aio api-mesh:log-list [-i] [--filename <value>] [--help]
+```
+
+### Flags
+
+The following arguments are optional:
+
+`--filename` allows you to download a CSV list of rayIDs with the specified filename. This flag exports all requests from the last 15 minutes.
+
+`-i` or `--ignoreCache` ignores the cached organization, project, and workspace, which allows you to make new selections. You can also manually [modify the cache](../basic/work-with-mesh.md#projects-and-workspaces).
+
+`--help` provides information on the specified command.
+
+### Response
+
+```terminal
+Selected organization: <ORG_NAME>
+Selected project: <PROJECT_NAME>
+Select workspace: <WORKSPACE_NAME>
+ Ray id           Timestamp      Response status Level          
+ ──────────────── ────────────── ─────────────── ────────────── 
+ 1a123456789abcd1 1724766278577  200             log            
+ 1a123456789abcd2 1724766287188  200             log            
+ 1a123456789abcd3 1724766286997  200             log            
+ 1a123456789abcd4 1724766280810  200             error
+ ```
+
+## aio api-mesh:log-get
+
+After finding the desired rayID with the [`aio api-mesh:log-list` command](#aio-api-meshlog-list), you can use the following command to retrieve the logs for a specific rayID:
+
+### Usage
+
+```bash
+aio api-mesh:log-get [RAYID] [-i] [--help]
+```
+
+### Flags
+
+The following arguments are optional.
+
+`-i` or `--ignoreCache` ignores the cached organization, project, and workspace, which allows you to make new selections. You can also manually [modify the cache](../basic/work-with-mesh.md#projects-and-workspaces).
+
+`--help` provides information on the specified command.
+
+#### Example
+
+The following example gets the log for the `1a123456789abcd0` rayID:
+
+```bash
+aio api-mesh:log-get 1a123456789abcd0
+```
+
+### Response
+
+```terminal
+EventTimestampMs : 1724660422500
+Exceptions : []
+Logs : [{'Level': 'log', 'Message': ['[object Object]'], 'TimestampMs': 1724660422580}, {'Level': 'log', 'Message': ['{"sources":[{"name":"venia","handler":{"graphql":{"useGETForQueries":true,"endpoint":"https://venia.magento.com/graphql","operationHeaders":{"x-test-header":"{context.headers[\'x-test-header\']}"}}}}],"responseConfig":{"includeHTTPDetails":true},"additionalResolvers":[],"plugins":[{"httpDetailsExtensions":{}}]}'], 'TimestampMs': 1724660422500}]
+Outcome : ok
+MeshId : 12a3b4c5-6d78-4012-3456-7e890fa1bcde
+RayId : 1a123456789abcd0
+MeshUrl : https://edge-graph.adobe.io/api/REDACTED/graphql
+RequestMethod : POST
+RequestStatus : 200 
+```
+
+## aio api-mesh:log-get-bulk
+
+The `log-get-bulk` command creates a CSV file with logs for the selected mesh during the specified time range. The maximum time range is 30 minutes and only logs from the last 30 days are accessible.
+
+### Usage
+
+```bash
+aio api-mesh:log-get-bulk [--startTime <value>] [--endTime <value>] [--filename <value>] [-i] [--help]
+```
+
+### Flags
+
+The following arguments are required:
```

Comment:
Technically, if the flag is required, they shouldn't be wrapped in braces `[]`, but the official usage statement wins.

Reply:
@keharper I consulted the output to begin with and have only used those outputs for the commands that already exist in the CLI.